storio-content-resolver/build.gradle

```diff
@@ -24,6 +24,7 @@ dependencies {
 
     provided rootProject.ext.supportAnnotations
     provided rootProject.ext.rxJava
+    provided rootProject.ext.rxAndroid
```

Comment:
Please remove this, `StorIOContentResolver` does no need `RxAndroid`, you can just remove `.observeOn(AndroidSchedulers.mainThread())` in tests because as I see, it does not matter for tests on which thread you are receiving results